modules/openapi-generator/src/main/resources/ruby-client/api_client_faraday_partial.mustache

```diff
@@ -10,10 +10,7 @@
         :client_cert => @config.ssl_client_cert,
         :client_key => @config.ssl_client_key
       }
-      request_options = {
-        :params_encoder => @config.params_encoder
-      }
-      connection = Faraday.new(:url => config.base_url, :ssl => ssl_options, :request => request_options) do |conn|
+      connection = Faraday.new(:url => config.base_url, :ssl => ssl_options) do |conn|
```

Comment:
@dkliban It seems the `params_encoder` option is configured again so I think we can remove it from here but let me know if my understanding is not correct. 

https://github.com/OpenAPITools/openapi-generator/blob/d17316e8d98be750fd87af528174b6584578d394/modules/openapi-generator/src/main/resources/ruby-client/api_client_faraday_partial.mustache#L87-L88